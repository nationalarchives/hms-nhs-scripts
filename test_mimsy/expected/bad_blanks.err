Warning: blank in "admission number" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "date of entry" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "name" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "quality" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "age" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "place of birth" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "port sailed out of" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "years at sea" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "last services" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "under what circumstances admitted (or nature of complaint)" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "date of discharge" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "how disposed of" at row 2 of input/bad_blanks.csv (volume 2, page 81)
Warning: blank in "number of days victualled" at row 2 of input/bad_blanks.csv (volume 2, page 81)
