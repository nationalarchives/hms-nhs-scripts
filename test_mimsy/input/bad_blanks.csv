,subject,volume,page,admission number,date of entry,name,quality,age,place of birth,port sailed out of,years at sea,last services,under what circumstances admitted (or nature of complaint),date of discharge,how disposed of,number of days victualled,Problems,Autoresolved,Repo,Commit,Args,§°—’“”…；£ªéºöœü
FAKE_URL,44121861,2,81,,,,,,,,,,,,,,,,b'origin\thttps://github.com/nationalarchives/hms-nhs-scripts.git (fetch)\norigin\thttps://github.com/nationalarchives/hms-nhs-scripts.git (push)\n',b'e045135190384b8107b6a9048c149549fa986f19\n',./aggregate.py -t 0.3,
