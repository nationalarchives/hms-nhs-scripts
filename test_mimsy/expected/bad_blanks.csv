DSH/2/81,admission number, ; date of entry, ; name, ; quality, ; age, ; place of birth, ; port sailed out of, ; years at sea, ; last services, ; under what circumstances admitted (or nature of complaint), ; date of discharge, ; how disposed of, ; number of days victualled, ;
